flowauth/backend/flowauth/config.py

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import logging
+
+import os
+from pathlib import Path
+from cryptography.fernet import Fernet
+
+
+def getsecret(key: str, default: str) -> str:
+    """
+    Get a value from docker secrets (i.e. read it from a file in
+    /run/secrets), return a default if the file is not there.
+
+    Parameters
+    ----------
+    key: str
+        Name of the secret.
+    default: str
+        Default value to return if the file does not exist
+
+    Returns
+    -------
+    str
+        Value in the file, or default
+    """
+    try:
+        with open(Path("/run/secrets") / key, "r") as fin:
+            return fin.read().strip()
+    except FileNotFoundError:
+        return default
+
+
+def get_config():
+    FLOWAUTH_FERNET_KEY = getsecret(
+        "FLOWAUTH_FERNET_KEY", os.environ["FLOWAUTH_FERNET_KEY"]
+    ).encode()
+    Fernet(FLOWAUTH_FERNET_KEY)  # Error if fernet key is bad
```

Comment:
Is it worth wrapping this call to `Fernet()` into a try/except block and essentially re-raise the error, just to make the error message a bit more explicit? Possibly not, just mentioning it because I always need to do a double take when I see a line of code that _looks_ like its purpose is to do something, but the only reason it exists is to throw an error.

If you don't want to go the explicit try/except route I suggest to at least explicitly ignore the return value so that it's apparent to the reader that this doesn't have any invisible side effects:
```python
_ = Fernet(FLOWAUTH_FERNET_KEY)  # Error if fernet key is bad
```